Cargo.toml

```diff
@@ -31,13 +31,14 @@ missing_debug_implementations = "warn"
 
 # We use this --cfg for documenting the cargo features on which an API
 # is available.  To preview this locally use: RUSTFLAGS="--cfg
-# iroh_docsrs cargo +nightly doc --all-features".  We use our own
+# iroh_docsrs" cargo +nightly doc --all-features.  We use our own
```

Comment:
?

Reply:
that's... the right way to write this command?
`RUSTFLAGS="--cfg iroh_docsrs cargo +nightly doc --all-features"` vs.
`RUSTFLAGS="--cfg iroh_docsrs" cargo +nightly doc --all-features`